spring-integration-syslog/src/main/java/org/springframework/integration/syslog/inbound/TcpSyslogReceivingChannelAdapter.java

```diff
@@ -36,9 +38,10 @@
 public class TcpSyslogReceivingChannelAdapter extends SyslogReceivingChannelAdapterSupport
 		implements TcpListener, ApplicationEventPublisherAware {
 
+	@SuppressWarnings("NullAway.Init")
 	private volatile AbstractServerConnectionFactory connectionFactory;
 
-	private volatile ApplicationEventPublisher applicationEventPublisher;
+	private volatile @Nullable ApplicationEventPublisher applicationEventPublisher;
```

Comment:
This one also `@SuppressWarnings("NullAway.Init")`.
Because it is initialized from the Application context callbacks.